api/python/provisioner/config.py

```diff
@@ -166,6 +166,14 @@
 CONTROLLER_B = 'b'
 CONTROLLER_BOTH = 'both'
 
+# configure_setup related Params
```

Comment:
I don't see a need for these enums.
If absolutely necessary, localize them to the `configure_setup.py` file

Reply:
they were used in both `configure_setup` and `validate` files, so have kept it in `config.py` for now. 
If their scope is not getting extended, will separately localize them to both files and use.